for_attendees.md

```diff
@@ -1,120 +1,127 @@
-# Remote Learning Thoughts for Workshop Attendees
+# Remote Learning Instructions for Workshop Attendees
 
 ### Introduction
 
-This document is an attempt to list all many points that need to be considered when attending
- a class that offers a remote option or is taught entirely remotely.  As in-person support is
-  not available, it is imperative that proper planning and sufficient lead time are given to
-   ensure a smooth and enjoyable class experience. Additionally, we&#39;d like to encourage
-    as much interaction as possible and when reasonable.
+This document lists points to consider when attending a workshop that offers a remote option
+or is taught entirely remotely. Most importantly, **ensure that all required software is
+installed and working correctly ahead of time**. As in-person support is not available, it is
+imperative that proper planning and sufficient lead time are given to ensure a smooth and
+enjoyable class experience. Additionally, we&#39;d like to encourage as much interaction as
+possible and when reasonable.
 
 For the purposes of this document, we will highlight solutions in [Zoom](http://zoom.us/),
- though likely solutions exist in other video conferencing software.
-
-We recommend the following guides on remote learning for reading:
-
- * [https://carpentries.github.io/instructor-training/guide/index.html#zoom](https://carpentries.github.io/instructor-training/guide/index.html#zoom)    PLEASE READ
-[https://teachremotely.harvard.edu/best-practices](https://teachremotely.harvard.edu/best-practices)
- * [http://www.russpoldrack.org/2020/01/talking-remotely-lessons-learned-so-far.html](http://www.russpoldrack.org/2020/01/talking-remotely-lessons-learned-so-far.html)
-  * [https://github.com/jfiksel/github-classroom-for-teachers](https://github.com/jfiksel/github-classroom-for-teachers)
+though solutions exist in other video conferencing software. A number of useful Zoom guides
+are available at [https://support.zoom.us/](https://support.zoom.us/).
 
```

Comment:
The guides are tricky because a) most are geared toward instructions, and b) most quickly go to instructions that don't apply to our workshops (e.g., starting Zoom from Canvas, software carpentry zoom accounts ...). Probably most of our attendees are pretty well familiar with Zoom by now...